docs/src/main/paradox/docs/fusion/search.md

```diff
@@ -1,62 +1,54 @@
-# Search
-
-Search provides functionality to search across all data in Nexus. Search is accessible via the Search menu option and the Search bar.
+# Global Types Search
 
 @@@ note
 
 Search requires Delta configuration. See the Delta Search documentation for details: @ref:[Delta Search Documentation](../delta/api/search-api.md)
 
 @@@
 
-## Search Bar
-
-The search bar at the top of screen provides both instant access to full-text search as well as convenient navigation 
-of Nexus. To activate, either click the search box or press the `/` key and begin typing.
+## Search Preset Cards
 
-![Search Bar](assets/fusion-search-bar.png)
+The search presets card at the home page allow the user to see the count of datasets for each type and select a specific type form the global types of Nexus. the same cards are also available on each global type page.
```

Comment:
* *allows* the user
* *from* the global types of Nexus.
* *The* same cards.... (i.e., uppercase t in The)